dt-metrics/combined/time-charts.js

```diff
@@ -0,0 +1,630 @@
+jQuery(function() {
```

Comment:
This file needs some window.lodash.escap()

Reply:
I think I got them all. I made sure to run them through my `escapeObject` function, which does all the escaping in one go e.g. lines 19-28